doc/pages/Install.md

```diff
@@ -60,114 +61,58 @@ sudo install <downloaded file> /usr/local/bin/opam
 
 ## Using your system's package manager
 
-This is generally the recommended way, **when available and up-to-date** (you
-can check [here](Distribution.html) the latest
-available release per distribution). Here is a list of supported distributions:
+This is generally the recommended way, **when available and up-to-date**. You
+can check the following table:
 
-#### Arch Linux
+[![repology package status](https://repology.org/badge/vertical-allrepos/opam.svg?exclude_unsupported=1)](https://repology.org/project/opam/versions)
```

Comment:
The list is quite long, i'm not sure that it should here but it's better than at the end i think...

Reply:
I think it makes much more sense at the beginning rather than the end.
The way users read it is:
1) what system do i have?
2) is the system package up-to-date?
  * If yes use that one (if you don't know how, the list would be right after)
  * If not, then the section just before on the binary installer will do

The per-distribution list is only useful for a minority of people (beginners), whereas the repology list is useful much more often in my opinion